### ISSUE TITLE
Code

### ISSUE THREAD

**Issue by Junjiewu2279** [DeepInsight-PCALab/ST-CGAN#4]
Hi! Thank you for the excellent work of the shadow removal, and where can i get the demo. The proposed ST-CGAN inspires me in other areas of research.

**Comment by jiahuan-wan** [comment]
The same, could you share the code and the trained model?

**Comment by XGBoost** [comment]
Where is the code???

**Comment by PabloK** [comment]
I'm also interested in having a look at the trained model.

**Comment by LetiP** [comment]
Where is the code? The model I implemented reading your paper will never converge. How did you do it?

**Comment by csjfwang** [comment]
Many thanks for your interests. We will share our code and trained model after our extensional study for this work.

**Comment by csjfwang** [comment]
@LetiP Thanks.
There are many reasons for your problem. Maybe you can change the learning rate or check your default setting where it is different from the description in our paper.

**Comment by hqhzz1993** [comment]
Hello,
    I think your work is excellent ,you are great!
    Recently, I'm interested in shadow removal ,and this paper inspires me. I think it's a masterpiece.
    I will grateful if you could offer me the code of paper .
     Thank you for reading this message , I hope your will get more great achievement in the future.

**Comment by arasharchor** [comment]
@csjfwang I am wondering whether this extensional study after one year! is still not finished. Please let us know if you are not going to publish the code.

**Comment by nachifur** [comment]
I think your work is excellent ! But this is CVPR2018 paper. Now 2020, When will the code be released？

**Comment by dfvr1994** [comment]
2022..no code released yet..
